client/components/account-balances/index.tsx

```diff
@@ -1,28 +1,265 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import { Card } from '@wordpress/components';
+import * as React from 'react';
+import { Flex, TabPanel } from '@wordpress/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import interpolateComponents from '@automattic/interpolate-components';
 
 /**
  * Internal dependencies
  */
-import AccountBalancesHeader from './header';
+import { useAllDepositsOverviews } from 'wcpay/data';
+import { useSelectedCurrency } from 'wcpay/overview/hooks';
+import { getCurrencyTabTitle } from './utils';
+import BalanceBlock from './balance-block';
+import BalanceTooltip from './balance-tooltip';
+import { documentationUrls, fundLabelStrings } from './strings';
+import InstantDepositButton from 'deposits/instant-deposits';
+import wcpayTracks from 'tracks';
 import './style.scss';
-import AccountBalancesTabPanel from './balances-tab-panel';
 
 /**
- * Renders an overview of the account's balances.
+ * BalanceTab
  *
- * @return {JSX.Element} Rendered element with the account balances card.
+ * @typedef {Object} BalanceTab
+ *
+ * @param {string} name           Name of the tab.
+ * @param {string} title          Title of the tab.
+ * @param {string} currencyCode   Currency code of the tab.
+ * @param {number} availableFunds Available funds of the tab.
+ * @param {number} pendingFunds   Pending funds of the tab.
+ * @param {number} delayDays	  The account's pending period in days.
+ */
```

Comment:
I see this type is used in a bunch of places – nice use of an interface / struct type to make this consistent and give it a name! Might be even clearer to add a summary/title – e.g. `Props used for displaying an account balance in a specific currency`, and/or name `BalanceTabProps`. (nitpick / not a blocker, this is fine as is)

Reply:
Thanks, renaming this helps to clarify it's meaning. I've renamed this to `BalanceTabProps` in 8c4c85fb08b1f26a3e206fde65b815abf6c15833